_drafts/meeting-announcement-template.md

```diff
@@ -0,0 +1,61 @@
+<!--
```

Comment:
Very useful template. I really like how we can choose a location to post for each announcement.